src/main/java/com/synopsys/integration/alert/channel/jira/JiraIssueHandler.java

```diff
@@ -186,12 +187,23 @@ public String createOrUpdateIssues(final FieldAccessor fieldAccessor, final Mess
                     }
                 }
             } catch (JiraMissingTransitionException e) {
-                missingTransitions.add(e.getTransition());
+                if (missingTransitionToIssues.containsKey(e.getTransition())) {
+                    missingTransitionToIssues.get(e.getTransition()).add(e.getIssueKey());
+                } else {
+                    Set<String> newIssueSet = new HashSet<>();
+                    newIssueSet.add(e.getIssueKey());
+                    missingTransitionToIssues.put(e.getTransition(), newIssueSet);
+                }
```

Comment:
We have this sort of functionality in multiple places around Alert (Map of string to set and adding to the set if string exists). I think we should create a wrapper class that performs this functionality with basic methods.

Reply:
Couldn't we use some of the new methods computeIfAbsent here?